dll/win32/shell32/CCopyToMenu.cpp

```diff
@@ -0,0 +1,116 @@
+#include "precomp.h"
+
+WINE_DEFAULT_DEBUG_CHANNEL(shell);
+
+CCopyToMenu::CCopyToMenu() :
+    m_pidlFolder(NULL),
+    m_idCmdFirst(0),
+    m_idCmdLast(0),
+    m_idCmdCopyTo(-1)
+{
+}
+
+CCopyToMenu::~CCopyToMenu()
+{
+    if (m_pidlFolder)
```

Comment:
Why not using a `CComHeapPtr<ITEMIDLIST>` here?

Reply:
OK.